index.html

```diff
@@ -85,6 +108,18 @@ <h2>
       Cases</a>
       </li>
     </ul>
+    <h3>
+      Multiple User Agents
```

Comment:
Feedback from 2024-12-11 call: user agent has a very specific meaning in the web platform. We are essentially trying "agent of the user". Either that say or find a new word.

Reply:
Also 2024-12-11 remove this for now and address it in its own section around roles / entities.